pages/common/pygmentize.md

```diff
@@ -1,27 +1,27 @@
 # pygmentize
 
-> Python based syntax highlighter.
+> Python-based syntax highlighter.
 
-- Highlight file syntax and print to standard output. Language is inferred from the file extension:
+- Highlight file syntax and print to standard output (language is inferred from the file extension):
 
 `pygmentize {{file.py}}`
 
 - Highlight syntax for a given language:
 
-`pygmentize -l {{javascript}} {{javascript_file}}`
+`pygmentize -l {{javascript}} {{input_file}}`
 
-- Show avaliable lexers:
+- List avaliable lexers (processors for input languages):
 
 `pygmentize -L lexers`
 
 - Save output to a file in HTML format:
 
-`pygmentize -f html -o {{file.html}} {{file.py}}`
+`pygmentize -f html -o {{output_file.html}} {{input_file.py}}`
 
-- Show avaliable output formats:
+- List avaliable output formats:
 
 `pygmentize -L formatters`
 
-- Output to HTML file with line numbers, specifying a given language:
+- Output a full (self-contained) HTML file, with line numbers:
 
-`pygmentize -f html -O linenos=1 -l {{language}} -o {{file.html}} {{file}}`
+`pygmentize -f html -O "full,linenos=True" -o {{output_file.html}} {{input_file}}`
```

Comment:
Hrm. I'm not sure that it could be done terribly concisely, unfortunately. You could extend the existing sentence to mention it, possible. Maybe something like "..., with line numbers, and specifying options....."? I'm not sure - I don't exactly have an experience with this utility myself :P